Tiltfile

```diff
@@ -33,7 +33,7 @@ settings = {
         "HCLOUD_REGION": "fsn1",
         "CONTROL_PLANE_MACHINE_COUNT": "3",
         "WORKER_MACHINE_COUNT": "3",
-        "KUBERNETES_VERSION": "v1.25.2",
+        "KUBERNETES_VERSION": "v1.27.3",
```

Comment:
is this correct?

Reply:
Yes, the stable version as per [kubernetes](https://www.downloadkubernetes.com/) for 1.27 is 1.27.8, but we don't have the kindest/node image with that tag. 
I used the latest tag that was present for 1.27. 
Ref: https://hub.docker.com/r/kindest/node/tags?page=1&name=v1.27